packages/charts/src/components/LineChart/LineChart.tsx

```diff
@@ -194,10 +198,12 @@ const LineChart: FC<LineChartProps> = forwardRef((props: LineChartProps, ref: Re
                 height={xAxisHeights[index]}
                 padding={xAxisPadding}
                 allowDuplicatedCategory={index === 0}
+                reversed={useIsRTL()}
```

Comment:
please don't call hooks inside props. extract it to `const isRTL = useIsRTL()` somewhere before the return and just use the value here.